packages/p2p/webpack.config.js

```diff
@@ -0,0 +1,90 @@
+const path = require('path');
+
+const is_serve   = process.env.BUILD_MODE === 'serve';
+const is_release = process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'staging';
+
+module.exports = {
+    entry: {
+        'index'  : path.resolve(__dirname, 'src/components', 'app.jsx'),
+    },
+    output: {
+        path         : path.resolve(__dirname, 'lib'),
+        filename     : '[name].js',
+        libraryExport: 'default',
+        library      : ['deriv-p2p', '[name]'],
```

Comment:
Why does the library name value have a dynamic `[name]` option?

Reply:
ill change it to inde.js